content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -1,59 +1,59 @@
 # Sharing Experiments
 
-There are two types of remotes that can store experiments. Git remotes are
-distributed copies of the Git repository, for example on GitHub or GitLab.
+If your team uses [Git remotes] on a Git server or hosting (e.g. GitHub, GitLab,
```

Comment:
what are the chances that teams do not use Git remotes? considering that we already mentioned that `dvc exp` won't work w/o Git?

Reply:
We mentioned that in a different page though, I think it's worth reiterating a little here for context. I rephrased it a little for clarity, PTAL.

Reply:
Rephrased to "DVC can rely on existing Git servers or hosting (e.g. GitHub, GitLab, etc.) to
save and share DVC Experiments." PTAL